### ISSUE TITLE
Not getting track titles via nuvolaplayer3ctl

### ISSUE THREAD

**Issue by scotte** [tiliado/nuvola-app-google-play-music#6]
I noticed that nuvolaplayer3ctl only has Artist, Album, and State, but no Title when playing Google play music. It seems to be an issue with this plugin, not nuvolaplayer3ctl. I don't know exactly why - it seems clear title is null (hence not displayed), perhaps there is no longer an element with id "player-song-title". I did find a div with it as data, so maybe it just needs to be queried out like "album" is?

```
<div id="currently-playing-title" class="tooltip">SONG_TITLE_HERE</div>
```

Assuming I am looking at the right part of the payload - a huge IF on my part. :-) Thanks!

**Comment by scotte** [comment]
Patch fixes now-playing data for nulvolaplayer3ctl and desktop notifications.

**Comment by fenryxo** [comment]
Hello. Thanks for you bug report and patch :+1:  Fixed in Google Play Music script 5.9.

**Comment by scotte** [comment]
Thanks!
